YACReader/main_window_viewer.cpp

```diff
@@ -1243,7 +1248,7 @@ void MainWindowViewer::processReset()
 void MainWindowViewer::setUpShortcutsManagement()
 {
     //actions holder
-    auto orphanActions = new QObject;
+    auto orphanActions = this;
```

Comment:
```suggestion
    auto orphanActions = new QObject(this);
```
The variable name @luisangelsm used here suggest he had a reason to set this as a plain QObject. We can fix the memleak easily by setting "this" as the QObject's parent but we still need confirmation why he intended this as an orphaned object.

Reply:
I haven't noticed any issues with using `this` (`MainWindowViewer`) as the actions' parent. There was an issue when I tried `viewer->addAction()`: the actions appeared in `Viewer`'s context menu. So I ended up calling [`this->`]`addAction()` instead.